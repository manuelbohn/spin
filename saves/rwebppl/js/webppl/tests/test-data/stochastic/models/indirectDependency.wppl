var f = function(x) {
  return (Math.pow(x, 3) + 5 * x) / 20;
}

var model = function() {
  var x = gaussian(0, 1);
  var y = gaussian(f(x), 0.5);
  return y;
};
