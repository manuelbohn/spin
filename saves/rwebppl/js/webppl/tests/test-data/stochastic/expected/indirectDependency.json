{
  "mean": 0,
  "std": 0.652
}
